doc/api/os.md

```diff
@@ -330,6 +330,18 @@ changes:
 Returns the operating system's default directory for temporary files as a
 string.
 
+On macOS, the returned path is usually a symlink. To get the resolved path, use
+[`fs.realpath()`][] or [`fs.realpathSync()`][].
+
+```js
+const tmpdirPath = os.tmpdir();
+
+// For most use cases, it does not matter whether tmpdirPath is a symlink or not
+// but if you need it to be a resolved path, use fs.realpath() or
```

Comment:
```suggestion
// but if a resolved path is needed , use fs.realpath() or
```

Reply:
Blech, passive voice, gross, I'm going with this instead:

```suggestion
// but to guarantee a resolved path, use fs.realpath() or
```